src/dbos-executor.ts

```diff
@@ -1586,9 +1586,29 @@ export class DBOSExecutor implements DBOSExecutorContext {
         this.logger.debug(`Skip local recovery because it's running in a VM: ${process.env.DBOS__VMID}`);
         continue;
       }
-      this.logger.debug(`Recovering workflows of executor: ${execID}`);
+      this.logger.debug(`Recovering workflows assigned to executor: ${execID}`);
       const wIDs = await this.systemDatabase.getPendingWorkflows(execID);
```

Comment:
@chuck-dbos AFAIK `getPendingWorkflows` is only used during recovery. I'd like to have it return more info from the status.